src/components/Composer/index.tsx

```diff
@@ -89,7 +89,7 @@ function Composer(
     const {windowWidth} = useWindowDimensions();
     const navigation = useNavigation();
     const textRef = useRef<HTMLElement & RNText>(null);
-    const textInput = useRef<HTMLTextAreaElement & TextInput>();
+    const textInput = useRef<(HTMLTextAreaElement & TextInput) | null>(null);
```

Comment:
```suggestion
    const textInput = useRef<HTMLTextAreaElement & TextInput>(null);
```

Same, I think we don't need `null` here.